pandas/core/reshape/tile.py

```diff
@@ -425,6 +426,7 @@ def _bins_to_cuts(
             labels = _format_labels(
                 bins, precision, right=right, include_lowest=include_lowest, dtype=dtype
             )
+            labels_are_unique = True
```

Comment:
Looking again, it may not be safe to assume that `_format_labels` returns a unique IntervalIndex like I thought. Might revert this for now.